extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3077,7 +3094,7 @@ static function getCityIDsFromVarValue( $varID, $val, $cond ) {
 			'cv_variable_id' => $varID
 		];
 
-		$dbr = self::db( DB_SLAVE );
+		$dbr = static::db( DB_SLAVE );
 
 		if ( in_array( $cond, [ 'LIKE', 'NOT LIKE' ] ) ) {
 			$aWhere[ ] = "cv_value " . str_replace( 'LIKE', '', $cond ) . $dbr->buildLike( $dbr->anyString(), serialize( $val ), $dbr->anyString() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "cv_value " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)